components/cdc/src/endpoint.rs

```diff
@@ -87,6 +90,7 @@ impl fmt::Debug for Deregister {
 }
 
 type InitCallback = Box<dyn FnOnce() + Send>;
+pub(crate) type OldValueCallback = Box<dyn FnMut(Key) -> Option<Vec<u8>> + Send>;
```

Comment:
Instead of naming it `Callback`, I think it's more like some `getter`. It's ok to me though.